src/Qir/Controller/QirController.csproj

```diff
@@ -1,12 +1,44 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Microsoft.Quantum.Qir</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <WarningLevel>4</WarningLevel>
```

Comment:
I am curious about why setting this warning level when the build configuration is Debug?

Reply:
I am not sure why but this was somehow added with the warning level as '5' which caused issues, so I downgraded it to '4'. If you don't think it is needed, I will remove it entirely. I'm not sure how it got added, though.